usaspending_api/database_scripts/matviews/functions_and_enums.sql

```diff
@@ -1,13 +1,13 @@
 -- The function(s) and enum(s) below were originally created for the materialized views
-
-CREATE TYPE total_obligation_bins AS ENUM ('<1M', '1M..25M', '25M..100M', '100M..500M', '>500M');
-CREATE TYPE award_type AS ENUM ('A', 'B', 'C', 'D', '02', '03', '04', '05', '06', '07', '08', '09', '10', '11');
-CREATE TYPE action_type AS ENUM ('b','H','S','C','L','J','W','V','N','Y','A','R',
-                                 'a','X','D','B','M','G','F','c','K','P','T','E');
-CREATE TYPE contract_pricing_type AS ENUM ('0','1','2','3','A','B','J','K','L','M',
-                                           'NO','O','P','R','S','T','U','V','X','Y','Z');
-
-
+--
+--CREATE TYPE total_obligation_bins AS ENUM ('<1M', '1M..25M', '25M..100M', '100M..500M', '>500M');
+--CREATE TYPE award_type AS ENUM ('A', 'B', 'C', 'D', '02', '03', '04', '05', '06', '07', '08', '09', '10', '11');
+--CREATE TYPE action_type AS ENUM ('b','H','S','C','L','J','W','V','N','Y','A','R',
+--                                 'a','X','D','B','M','G','F','c','K','P','T','E');
+--CREATE TYPE contract_pricing_type AS ENUM ('0','1','2','3','A','B','J','K','L','M',
+--                                           'NO','O','P','R','S','T','U','V','X','Y','Z');
+--
+--
```

Comment:
Is this no longer required?

Reply:
It is required, good catch. Accidentally left commented. Will fix.